core/src/main/scala/com/softwaremill/react/kafka/ReactiveKafkaConsumer.scala

```diff
@@ -15,7 +32,18 @@ case class ReactiveKafkaConsumer[K, V](properties: ConsumerProperties[K, V]) {
       properties.keyDeserializer,
       properties.valueDeserializer
     )
-    c.subscribe(List(properties.topic)) // support multiple topics?
+
+    if (topicPartitionOffset.isInstanceOf[TopicPartitionOffset]) {
+      c.seek(new TopicPartition(topicPartitionOffset.topic, topicPartitionOffset.partition), topicPartitionOffset.offset)
```

Comment:
Can't we call `seek` multiple times, each time for different t+p+o? This way we could only require a `List[TopicPartitionOffset]` in the API.